pypika/tests/dialects/test_postgresql.py

```diff
@@ -207,3 +207,14 @@ def test_json_has_any_keys(self):
         self.assertEqual(
             "SELECT * FROM \"abc\" WHERE \"json\"?|ARRAY['dates','imported']", str(q)
         )
+
+
+class DistinctONTests(unittest.TestCase):
```

Comment:
very minor, but noticed so will point it out, the N in On here should perhaps be lowercased.